stock_picking_report_external_note/report/report_deliveryslip.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) -->
+<odoo>
+    <template
+        id="report_delivery_sticker_document"
```

Comment:
Can you change the id of the view? It's the same of another module (I forgot to change it...)

```suggestion
        id="report_external_note_delivery_document"
```